site/content/docs/reference/audit-logging.md

```diff
@@ -55,47 +55,48 @@ These APIs include:
   The token endpoint can handle the standard OIDC `authorization_code` and `refresh_token` grant types, and has also been
   extended to handle an additional grant type for [RFC 8693](https://datatracker.ietf.org/doc/html/rfc8693) token exchanges to
   reduce the applicable scope (technically, the `aud` claim) of ID tokens.
-- `<issuer_path>/callback` is a special endpoint that is used as the redirect URL when performing an OAuth 2.0 or OIDC authcode flow against an upstream OIDC identity provider as configured by an OIDCIdentityProvider or GitHubIdentityProvider custom resource.
+- `<issuer_path>/callback` is a special endpoint that is used as the redirect URL when performing an OAuth 2.0 or OIDC authcode flow against an upstream OIDC identity provider as configured by an `OIDCIdentityProvider` or `GitHubIdentityProvider` custom resource.
 - `<issuer_path>/login` is a login UI page to support the optional browser-based login flow for LDAP and Active Directory identity providers.
 
 ## Structure of an audit event
 
-Every line in a Supervisor or Concierge pod log is a JSON object. Only the lines that include the
+Every log line in a Supervisor or Concierge pod log is a JSON object. Only those log lines that include the
 key/value pair `"auditEvent": true` are audit events. Other lines are for errors, warnings, and
 debugging information.
 
-Every line contains the following keys, and audit event lines also contain these common keys/values:
+Every audit log contains the following keys, and audit event lines also contain these common keys/values:
```

Comment:
@joshuatcasey Small correction: Every line in the whole log includes these common keys, not only audit events.

Reply:
Fixed!